### PR TITLE
Add 2019-06-01-preview API version for Service Fabric

### PR DESCRIPTION
**Pull request by linmeng08**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by linmeng08** [comment]
Hi @sarangan12, @samedder - can any of you please take a look at this PR?

File: specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-06-01-preview/cluster.json [hunk 1]
```diff
@@ -0,0 +1,1538 @@
+{
```
Comment (KrisBash) [inline specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-06-01-preview/cluster.json]: For new API versions, we much prefer that the first commit be an unchanged copy of the last reviewed API version. Then subsequent commits can reflect the necessary changes. This facilitates simpler review of the changes in GH.

**Review by KrisBash — APPROVED** [review]
Changes LGTM

File: specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-06-01-preview/cluster.json (hunk 1 above)
Comment (linmeng08) [reply specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-06-01-preview/cluster.json]: I see - thanks, that makes a lot of senses. We will update the doc on our end to add this step for any new API changes going forward.

**Comment by yungezz** [comment]
CI passed in Azure Pipeline https://github.com/Azure/azure-rest-api-specs/pull/7092/checks?check_run_id=208525848.

**Comment by yungezz** [comment]
hi @linmeng08 is this PR ready to merge?

**Comment by linmeng08** [comment]
> hi @linmeng08 is this PR ready to merge?

Yes please proceed, thank you!

**Review by yungezz — APPROVED** [review]

**Pull request merged**